evm/gateway.js

```diff
@@ -183,6 +195,38 @@ async function processCommand(config, chain, options) {
             break;
         }
 
+        case 'approveWithBatch': {
+            const { batchID, api } = options;
```

Comment:
there's no flag for the api. Reading it from the config should be fine, don't have to add an override for options where we don't have a need for it yet